packages/module/src/MessageDivider/MessageDivider.scss

```diff
@@ -0,0 +1,59 @@
+// ============================================================================
+// Chatbot Main - Message Divider
+// ============================================================================
+.pf-chatbot__message-divider {
+  padding: 12px 0;
+}
+
+.pf-chatbot__message-divider-content {
+  display: flex;
+  height: 24px;
+  padding: 0px var(--pf-t--global--spacer--md);
+  justify-content: center;
+  align-items: center;
+  gap: 8px;
+  border-radius: var(--pf-t--global--border--radius--pill);
+  background: var(--pf-t--global--background--color--secondary--default);
```

Comment:
This is my fault - I should have noticed this in planning - but we will need to get a new color from Kayla. This is the same color as the ChatBot background, so in context, the background doesn't work:

<img width="379" height="571" alt="Screenshot 2025-07-10 at 1 00 20 PM" src="https://github.com/user-attachments/assets/6b9685c5-1688-448d-bff0-abe733b34201" />

We probably need to use global--background--color--tertiary--default, but let's confirm with Kayla. Figma would need to change as well.